.cirrus.yml

```diff
@@ -224,7 +234,7 @@ task:
     ### iOS+macOS tasks ***
     - name: lint_darwin_plugins
       script:
-        - ./script/tool_runner.sh podspecs
+        - dart $PLUGIN_TOOL podspecs $COMMON_TOOL_FLAGS
```

Comment:
> I don't think this is really an improvement because now we've introduced the risk of someone forgetting COMMON_TOOL_FLAGS.

Yes, I went back and forth on this for the same reason, and didn't see a good solution. The main benefit is that it eliminates the strange setup where `tool_runner.sh` is reading a magic env variable, and `.cirrus.yml` sets the magic variable, but it's totally non-obvious in each one in isolation what this variable is.

But I'll just revert this part for now and revisit it later.

> If you made the command being run not have to be positional with the presence of a new flag you could extract `RUN_PLUGIN_TOOL=dart $PLUGIN_TOOL $COMMON_TOOL_FLAGS --command `

I looked into this a bit, and AFAICT allowing flags to be before the command with `CommandRunner` requires declaring them at different level that has totally different (and much more awkward) affordances for reading the flags. I can probably move just this one flag as a hack, but it seems ugly. Maybe I'll find a better solution when revisiting.